core/ir/opnd.h

```diff
@@ -2756,10 +2756,20 @@ DR_API
 /**
  * Assumes that both \p old_reg and \p new_reg are DR_REG_ constants.
  * Replaces all occurrences of \p old_reg in \p *opnd with \p new_reg.
+ * Returns whether it replaced anything.
```

Comment:
Maybe clarify explicitly that `opnd_replace_reg` will not replace size variants, and point to `opnd_replace_reg_resize`.

Reply:
Done.